crypto/ec/ec_lib.c

```diff
@@ -66,13 +66,13 @@ EC_GROUP *EC_GROUP_new(const EC_METHOD *meth)
 void EC_pre_comp_free(EC_GROUP *group)
 {
     switch (group->pre_comp_type) {
-    default:
+    case PCT_none:
         break;
-#ifdef ECP_NISTZ256_REFERENCE_IMPLEMENTATION
     case PCT_nistz256:
+#ifdef ECP_NISTZ256_REFERENCE_IMPLEMENTATION
```

Comment:
Comment in ecp_nistz256.c and macro occurrence there are fine. Way to clarify it is to imagine that ECP_NISTZ256_REFERENCE_IMPLEMENTATION was called ECP_NISTZ256_REFERENCE_IMPLEMENTATION_THAT_IS_MEANIGFUL_TO_DEFINE_ONLY_IN_ECP_NISTZ256_C_ITSELF.